tools/flaky_test_monitor/common/config.go

```diff
@@ -0,0 +1,76 @@
+package common
+
+import (
+	"bufio"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// Properties used in property file
+const FailureThresholdPercent = "failures_threshold_percent"
+const FailuresSliceMax = "failures_slice_max"
+
+const DurationThresholdSeconds = "duration_threshold_seconds"
+const DurationSliceMax = "duration_slice_max"
+
+// const Duration
+
+type Config struct {
+	FailureThresholdPercent float32
+	FailuresSliceMax        int
+
+	DurationThresholdSeconds float32
+	DurationSliceMax         int
+}
+
+func ReadProperties(directory string) Config {
+	// looks for properties file with specific name
+	propertyFile := AddTrailingSlash(directory) + "flaky-test-monitor.properties"
+
+	file, err := os.Open(propertyFile)
+
+	AssertNoError(err, "error opening property file")
+	defer file.Close()
+
+	properties := map[string]string{}
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
```

Comment:
a little surprised we decided to write our own parser here, thought there'd be some lib we'd be able to use for this.

Reply:
Yeah, I thought about using a lib, but I didn't want the overhead. This property file parser is only about 15 lines if you just take into account the "generic parsing" it's doing. It's also doing a bunch of Flaky Test Monitor specific parsing which we'd need to do with an off-the-shelf library, too.